packages/cli/src/constants.ts

```diff
@@ -107,3 +107,10 @@ export enum CLILogLevel {
 
 export const sqlPasswordQustionName = "sql-password";
 export const sqlPasswordConfirmQuestionName = "sql-confirm-password";
+export const booleanParamNames = new Set<string>([
```

Comment:
If we do this, all newly added boolean options need to update this array, right?

Reply:
Yes. if not, there will be error if user input capital case parameter name. (but if user input lowercase param name, yargs will help parse it and all will be fine.)

Reply:
Can we find another way to judge if an option is a Boolean option? Just don't wanna introduce another hard  code here. Let's discuss it.